.eslintrc.cjs

```diff
@@ -50,6 +50,15 @@ module.exports = {
         'includeRoles': ['button']
       },
     ],
+    '@typescript-eslint/no-unused-vars': [
+      1,
+      {
+        vars: 'all',
+        varsIgnorePattern: '^_',
+        args: 'after-used',
+        argsIgnorePattern: '^_',
```

Comment:
we might want
eqeqeq: To enforce the use of === and !== over == and !=.

abstract equality operators can lead to human error
0 == '0' is true but 0 === '0' is false. ignores type in this case which can lead to logical error.

Reply:
I did not understand. This is a rule with some additional settings for the no-unused-vars rule.

```ts
function exampleFunction(_unusedArg: string, usedArg: number) {
  const _unusedVar = 'I am not used';
  console.log(usedArg);
}
```
In this example, the `_unusedArg` and `_unusedVar` are prefixed with an underscore, so they are ignored by the no-unused-vars rule as specified in the configuration.

Reply:
Are you asking to add a lint rule for using `===`?

Reply:
oh i wanted to hear your thoughts on adding a lint rule for using ===. Personally, I find using both === and == confusing and personally I find not using == is fine.

Reply:
right. I completely agree with the use of `===`. I will add the rule.

Reply:
if we are alreadying using == in the project and not so simple to replace it, we can take this separate task